fvtest/jitbuildertest/SelectTest.cpp

```diff
@@ -0,0 +1,185 @@
+   {
+   OMR::JitBuilder::IlValue * condition = Load("condition");
+   OMR::JitBuilder::IlValue * trueValue = Load("trueValue");
+   OMR::JitBuilder::IlValue * falseValue = Load("falseValue");
+   OMR::JitBuilder::IlValue * result = Select(condition, trueValue, falseValue);
+   Return(result);
+   return true;
+   }
+
+DEFINE_BUILDER(TestInt32Select,
+               Int32,
+               PARAM("condition", Int32),
+               PARAM("trueValue", Int32),
+               PARAM("falseValue", Int32))
+   {
+   OMR::JitBuilder::IlValue * condition = Load("condition");
+   OMR::JitBuilder::IlValue * trueValue = Load("trueValue");
+   OMR::JitBuilder::IlValue * falseValue = Load("falseValue");
+   OMR::JitBuilder::IlValue * result = Select(condition, trueValue, falseValue);
+   Return(result);
+   return true;
+   }
+
+DEFINE_BUILDER(TestInt64Select,
+               Int64,
+               PARAM("condition", Int32),
+               PARAM("trueValue", Int64),
+               PARAM("falseValue", Int64))
+   {
+   OMR::JitBuilder::IlValue * condition = Load("condition");
+   OMR::JitBuilder::IlValue * trueValue = Load("trueValue");
+   OMR::JitBuilder::IlValue * falseValue = Load("falseValue");
+   OMR::JitBuilder::IlValue * result = Select(condition, trueValue, falseValue);
+   Return(result);
+   return true;
+   }
+
+DEFINE_BUILDER(TestAddressSelect,
+               Address,
+               PARAM("condition", Int32),
+               PARAM("trueValue", Address),
+               PARAM("falseValue", Address))
+   {
+   OMR::JitBuilder::IlValue * condition = Load("condition");
+   OMR::JitBuilder::IlValue * trueValue = Load("trueValue");
+   OMR::JitBuilder::IlValue * falseValue = Load("falseValue");
+   OMR::JitBuilder::IlValue * result = Select(condition, trueValue, falseValue);
+   Return(result);
+   return true;
+   }
+
+DEFINE_BUILDER(TestFloatSelect,
+               Float,
+               PARAM("condition", Int32),
+               PARAM("trueValue", Float),
+               PARAM("falseValue", Float))
+   {
+   OMR::JitBuilder::IlValue * condition = Load("condition");
+   OMR::JitBuilder::IlValue * trueValue = Load("trueValue");
+   OMR::JitBuilder::IlValue * falseValue = Load("falseValue");
+   OMR::JitBuilder::IlValue * result = Select(condition, trueValue, falseValue);
+   Return(result);
+   return true;
+   }
+
+DEFINE_BUILDER(TestDoubleSelect,
+               Double,
+               PARAM("condition", Int32),
+               PARAM("trueValue", Double),
+               PARAM("falseValue", Double))
+   {
+   OMR::JitBuilder::IlValue * condition = Load("condition");
+   OMR::JitBuilder::IlValue * trueValue = Load("trueValue");
+   OMR::JitBuilder::IlValue * falseValue = Load("falseValue");
+   OMR::JitBuilder::IlValue * result = Select(condition, trueValue, falseValue);
+   Return(result);
+   return true;
+   }
+
+class SelectTest : public JitBuilderTest {};
+
+typedef int8_t (*Int8ReturnType)(int32_t,int8_t,int8_t);
```

Comment:
What about using a name like `Int8TestFunctionType`, since the `Int8` doesn't just apply to the return type?